apps/settings/test/marionette/keyboardtestapp/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Keyboard APP Test</title>
+  </head>
+  <body>
+    <div>
+      <input type="text" id="textInput"/>
```

Comment:
I tried including it from the keyboard test directory but couldn't get it working. Maybe it's just some syntax problem. Will try again.